huxley/www/js/components/AdvisorRosterView.js

```diff
@@ -168,10 +170,31 @@ var AdvisorRosterView = React.createClass({
             </Button>
           </td>
         );
+
+        var waiverCheck = '';
+        if (delegate && delegate.waiver_submitted) {
+          waiverCheck = '\u2611';
+        } else {
+          waiverCheck = '\u2610';
+        }
+
+        var positionPaperCheck = '';
+        if (
+          delegate.assignment &&
```

Comment:
If you guys update Babel, you can use the optional property access operator to make this much shorter: https://www.npmjs.com/package/babel-plugin-transform-optional-chaining